Packages/vcs/Lib/vcs2vtk.py

```diff
@@ -1182,29 +1182,38 @@ def prepMarker(renWin,marker,cmap=None):
   return actors
 
 def prepLine(renWin,line,cmap=None):
-  n = prepPrimitive(line)
-  if n==0:
+  number_lines = prepPrimitive(line)
+
+  if number_lines == 0:
     return
+
   actors = []
-  for i in range(n):
+
+  for i in range(number_lines):
     l = vtk.vtkLine()
     lines = vtk.vtkCellArray()
     x = line.x[i]
-    y=line.y[i]
-    c=line.color[i]
-    w=line.width[i]
-    t=line.type[i]
-    N = max(len(x),len(y))
+    y = line.y[i]
+    c = line.color[i]
+    w = line.width[i]
+    t = line.type[i]
+    number_points = max(len(x),len(y))
+
+    # Extend x or y to the length of the other by duplicating the last coord.
     for a in [x,y]:
-      while len(a)<n:
+      while len(a)<number_points:
         a.append(a[-1])
```

Comment:
@doutriaux1 This looks like the same issue we had in 46105aafee32a0657f49859d28cf56952eb41cc4. Can you check the similar functions in this file and make sure they're treating both n and N correctly?

I think this `while len(a) < number_points: a.append(a[-1])` bit should just be `assert(len(a) == number_points)`(or similar failure) here, too, unless this is an expected feature. Otherwise it would likely just mask user mistakes and give unexpected incorrect results.